rsvg-convert missing from 21.02

## Describe the bug

20.02 and earlier had `rsvg-convert`, which _something_ (pandoc or xelatex), needs to convert SVG. 21.02 does not have it.

```
$ docker run -it --rm --entrypoint bash dalibo/pandocker:20.02
root@b80c6e089c2b:/pandoc# command -v rsvg-convert
/usr/bin/rsvg-convert
root@b80c6e089c2b:/pandoc#
```

```
$ docker run -it --rm --entrypoint bash dalibo/pandocker:21.02
root@1e3eca8864e1:/pandoc# command -v rsvg-convert
root@1e3eca8864e1:/pandoc#
```

## Steps To Reproduce

Convert a Markdown file that references an SVG image.

```
[WARNING] Could not convert image '/tmp/tex2pdf.-3d2f4ac3f30cd44a/a2bba2e68fa6fa402a11d0687787345f9786e291.svg': check that rsvg-convert is in path.
  rsvg-convert: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
[WARNING] Could not convert image '/tmp/tex2pdf.-3d2f4ac3f30cd44a/a2bba2e68fa6fa402a11d0687787345f9786e291.svg': check that rsvg-convert is in path.
  rsvg-convert: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
Error producing PDF.
! LaTeX Error: Cannot determine size of graphic in /tmp/tex2pdf.-3d2f4ac3f30cd4
4a/a2bba2e68fa6fa402a11d0687787345f9786e291.svg (no BoundingBox).

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.234 ...a2e68fa6fa402a11d0687787345f9786e291.svg}

make: *** [whitepaper-template-example.pdf] Error 43
```

## Expected Behavior

pandocker's pandoc should handle SVG. It cannot do so without rsvg-convert being available.

--- a/alpine/Dockerfile
+++ b/alpine/Dockerfile
@@ -7,7 +7,8 @@ RUN apk --no-cache add \
         openssh-client \
         fontconfig \
         ttf-liberation \
-        dia
+        dia \
+        librsvg
 
 RUN tlmgr update --self && \
     tlmgr install \
@@ -31,7 +32,7 @@ RUN tlmgr update --self && \
         hyphen-dutch \
         #lang-german: https://packages.debian.org/sid/texlive-lang-german
         babel-german \
-        xpatch \ 
+        xpatch \
         #lang-spanish: https://packages.debian.org/unstable/texlive-lang-spanish
         babel-spanish \
         hyphen-spanish \

--- a/buster-full/Dockerfile
+++ b/buster-full/Dockerfile
@@ -41,7 +41,7 @@ RUN set -x && \
         fonts-noto-core \
         fonts-noto-extra \
         fonts-noto-mono \
-        # for emojis
+        # for emojis and SVG conversion
         librsvg2-bin \
     # clean up
     && apt-get clean && \

--- a/buster/Dockerfile
+++ b/buster/Dockerfile
@@ -62,6 +62,8 @@ RUN set -x && \
         # required for PDF meta analysis
         poppler-utils \
         zlibc \
+        # svg conversion
+        librsvg2-bin \
     # clean up
     && apt-get clean && \
     rm -rf /var/lib/apt/lists/* /etc/apt/apt.conf.d/01proxy

--- a/stretch/Dockerfile
+++ b/stretch/Dockerfile
@@ -64,7 +64,7 @@ RUN set -x && \
         # required for PDF meta analysis
         poppler-utils \
         zlibc \
-        # for emojis
+        # for emojis and SVG conversion
         librsvg2-bin \
     # clean up
     && apt-get clean && \

